lib/helper.c

```diff
@@ -1026,3 +1026,40 @@ ipmi_get_oem_id(struct ipmi_intf *intf)
 
 	return oem_id;
 }
+
+uint16_t
+ipmi_get_qct_paltform_id(struct ipmi_intf *intf)
```

Comment:
This doesn't belong here. It must go to `lib/ipmi_quantaoem.c` just like it is done for sunoem, kontronoem, et al.